angular 14.2 (and typescript 4.8.x) - schema-validator.service.ts (and ngx-charts)

I try to update ngx-charts for angular 14.2 (and typescript 4.8.4).
When I compile ngx-charts, I have an error with ngx-ui:
```
Error: node_modules/@swimlane/ngx-ui/lib/components/json-editor/schema-validator.service.d.ts:4:10 - error TS2702: 'Ajv' only refers to a type, 
but is being used as a namespace here.
4     ajv: Ajv.Ajv;
```
If i patch node_modules\@swimlane\ngx-ui\lib\components\json-editor\schema-validator.service.d.ts with:
BEFORE: `export declare class SchemaValidatorService {   ajv: Ajv.Ajv;`
AFTER: `export declare class SchemaValidatorService {   ajv: Ajv;`
all is ok.

**With this error, it is not possible to post a pull request for ngx-charts for angular 14.2.**

Is-it possible to change the code for ngx-ui/projects/swimlane/ngx-ui/src/lib/components/json-editor/schema-validator.service.ts
like this (for all typescript version) => declare variable with constructor
BEFORE:
```
export class SchemaValidatorService {
  ajv: Ajv.Ajv;

  constructor() {
    this.ajv = new Ajv({
      allErrors: true
    });
    this.ajv.addFormat('password', '.*');
    this.ajv.addFormat('code', '.*');
    this.ajv.addFormat('binary', '.*');
  }
```
AFTER:
```
export class SchemaValidatorService {
  ajv = new Ajv({
      allErrors: true
    });

  constructor() {
    this.ajv.addFormat('password', '.*');
    this.ajv.addFormat('code', '.*');
    this.ajv.addFormat('binary', '.*');
  }
```

Philippe

--- a/projects/swimlane/ngx-ui/src/lib/components/json-editor/schema-validator.service.ts
+++ b/projects/swimlane/ngx-ui/src/lib/components/json-editor/schema-validator.service.ts
@@ -5,12 +5,11 @@ import { Injectable } from '@angular/core';
   providedIn: 'root'
 })
 export class SchemaValidatorService {
-  ajv: Ajv.Ajv;
+  ajv = new Ajv({
+    allErrors: true
+  });
 
   constructor() {
-    this.ajv = new Ajv({
-      allErrors: true
-    });
     this.ajv.addFormat('password', '.*');
     this.ajv.addFormat('code', '.*');
     this.ajv.addFormat('binary', '.*');

